### PR TITLE
Fixed plugins_url and https

### PR DESCRIPTION
**Pull request by pur4u**
Fixed plugins_url and https.

### CODE REVIEW

File: cf-admin.php [hunk 1]
```diff
@@ -26,11 +26,15 @@ static function path_to_adminui() {
 		}
 		
 		static function url_to_adminui() {
+			if ( strpos( CF_ADMIN_DIR, plugins_url() ) !== false ) {
+				return CF_ADMIN_DIR;
+			}
+
 			if (is_dir(trailingslashit(WP_PLUGIN_DIR).CF_ADMIN_DIR)) {
-				return trailingslashit(WP_PLUGIN_URL).CF_ADMIN_DIR;
+				return  preg_replace('#^https?://#i', '//', trailingslashit(WP_PLUGIN_URL).CF_ADMIN_DIR);
```
Comment (mbucurcf) [inline cf-admin.php]: Fix spacing.

File: cf-admin.php [hunk 2]
```diff
@@ -26,11 +26,15 @@ static function path_to_adminui() {
...
 			}
 			else if (is_dir(trailingslashit(TEMPLATEPATH).'plugins/'.CF_ADMIN_DIR)) {
-				return trailingslashit(get_bloginfo('template_url')).'plugins/'.CF_ADMIN_DIR;
+				return  preg_replace('#^https?://#i', '//', trailingslashit(get_bloginfo('template_url')).'plugins/'.CF_ADMIN_DIR);
```
Comment (mbucurcf) [inline cf-admin.php]: Fix spacing.

File: cf-admin.php [hunk 3]
```diff
@@ -297,4 +301,4 @@ static function activate_plugin_for_new_blog($file, $blog_id, $single_activation
 		}
 	}
 
-} // end class-exists check ?>
\ No newline at end of file
+} // end class-exists check ?>
```
Comment (mbucurcf) [inline cf-admin.php]: Comment should start with uppercase and end with a dot.
Also, you can remove the ending PHP tag `?>`, if it's at the end of the file.

**Review by bucur-marian — CHANGES_REQUESTED** [review]

File: cf-admin.php [hunk 4]
```diff
@@ -14,31 +14,29 @@
...
 			}
+
 			return false;
 		}
-		
+
 		static function url_to_adminui() {
 			if ( strpos( CF_ADMIN_DIR, plugins_url() ) !== false ) {
 				return CF_ADMIN_DIR;
+			} else if ( is_dir( trailingslashit( WP_PLUGIN_DIR ) . CF_ADMIN_DIR ) ) {
```
Comment (mbucurcf) [inline cf-admin.php]: Use `elseif` instead of `else if`.

File: cf-admin.php [hunk 5]
```diff
@@ -14,31 +14,29 @@
...
+
 		static function path_to_adminui() {
-			if (is_dir(trailingslashit(WP_PLUGIN_DIR).CF_ADMIN_DIR)) {
-				return trailingslashit(WP_PLUGIN_DIR).CF_ADMIN_DIR;
-			}
-			else if (is_dir(trailingslashit(TEMPLATEPATH).'plugins/'.CF_ADMIN_DIR)) {
-				return trailingslashit(TEMPLATEPATH).'plugins/'.CF_ADMIN_DIR;
+			if ( is_dir( trailingslashit( WP_PLUGIN_DIR ) . CF_ADMIN_DIR ) ) {
+				return trailingslashit( WP_PLUGIN_DIR ) . CF_ADMIN_DIR;
+			} else if ( is_dir( trailingslashit( TEMPLATEPATH ) . 'plugins/' . CF_ADMIN_DIR ) ) {
```
Comment (mbucurcf) [inline cf-admin.php]: Use `elseif` instead of `else if`.

File: cf-admin.php [hunk 6]
```diff
@@ -14,31 +14,29 @@
...
 			return false;
 		}
-		
+
 		static function url_to_adminui() {
 			if ( strpos( CF_ADMIN_DIR, plugins_url() ) !== false ) {
 				return CF_ADMIN_DIR;
+			} else if ( is_dir( trailingslashit( WP_PLUGIN_DIR ) . CF_ADMIN_DIR ) ) {
+				return preg_replace( '#^https?://#i', '//', trailingslashit( WP_PLUGIN_URL ) . CF_ADMIN_DIR );
+			} else if ( is_dir( trailingslashit( TEMPLATEPATH ) . 'plugins/' . CF_ADMIN_DIR ) ) {
```
Comment (mbucurcf) [inline cf-admin.php]: Use `elseif` instead of `else if`.

**Review by bucur-marian — APPROVED** [review]
+1

**Pull request merged**